misc/python/materialize/scalability/regression.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from materialize.scalability.endpoint import Endpoint
+
+
+class Regression:
+    def __init__(
+        self,
+        workload_name: str,
+        concurrency: int,
+        count: int,
+        tps: float,
+        tps_baseline: float,
+        tps_diff: float,
+        tps_diff_percent: float,
+        endpoint: Endpoint,
+    ):
+        self.workload_name = workload_name
+        self.concurrency = concurrency
+        self.count = count
+        self.tps = tps
+        self.tps_baseline = tps_baseline
+        self.tps_diff = tps_diff
```

Comment:
assert that tps_diff is always negative here

Reply:
✅